packages/tools/fetch-tool/src/fluidFetchSnapshot.ts

```diff
@@ -146,8 +146,8 @@ async function fetchBlobsFromSnapshotTree(
             console.error(`No data store tree for data store = ${dataStore}, path = ${prefix}, version = ${dataStoreVersions[0].id}`);
             continue;
         }
-        assert(dataStoreSnapShotTree.id === tree.commits[dataStore]);
-        assert(dataStoreSnapShotTree.id === dataStoreVersions[0].id);
+        assert(dataStoreSnapShotTree === undefined || dataStoreSnapShotTree.id === tree.commits[dataStore]);
```

Comment:
Not too familiar with the code, should this be?
```suggestion
        assert(dataStoreSnapShotTree.id === undefined || dataStoreSnapShotTree.id === tree.commits[dataStore]);
```